sdks/python/scripts/run_whitespacelint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+set -o errexit
+
+pushd ../../
+
+# Check for trailing spaces in non-code text files.
+# Currently include markdown files and gradle build files.
+echo "Running whitespacelint..."
+FILES=$(find .  \( -name '*.md' -o -name 'build.gradle' \))
+for FILE in $FILES
+do
+  whitespacelint $FILE
```

Comment:
I suggest `whitespacelint "$FILE"` to correctly handle whitespaces and prevent globbing.

Reply:
Thanks, changing it.